### PR TITLE
[CP-498] Implementing file system dialog functionality

### PR DESCRIPTION
**Pull request by igorbezsmertnyi**
Jira: [CP-498]

**Description**
Implementing file system dialog functionality

<details>
<summary><b>Screenshots</b></summary>
// put images here
</details>

[CP-498]: https://appnroll.atlassian.net/browse/CP-498?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/app/src/file-system/constants/error.constant.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+export enum DialogFileSystemError {
+  GetPath = "CANT_GET_SELECTED_FILES_PATH",
```
Comment (dkarski) [inline packages/app/src/file-system/constants/error.constant.ts:7]: New error description strategy? 😉

**Review by dkarski — DISMISSED** [review]
✅ LGTM

File: packages/app/src/file-system/dto/get-paths.input.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { FileFilter, OpenDialogOptions } from "electron"
+
+export interface GetPathInput {
```
Comment (wojcik92michal) [inline packages/app/src/file-system/dto/get-paths.input.ts:8]: Shouldn't it be called `GetPathsInput` (`Paths` vs `Path`) and the file be called something like `get-paths-input.object.ts` (just to follow the convention from other modules)?

File: packages/app/src/file-system/requests/get-paths.request.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { ipcRenderer } from "electron-better-ipc"
+import { ResultObject } from "App/core/builder"
+import { IpcDialogFileSystemRequest } from "App/file-system/constants"
+import { GetPathInput } from "App/file-system/dto"
+
+export const getPathRequest = async (
```
Comment (wojcik92michal) [inline packages/app/src/file-system/requests/get-paths.request.ts:11]: Seems to me that this should be `getPathsRequest`

**Review by wojcik92michal — DISMISSED** [review]

File: packages/app/src/file-system/requests/get-paths.request.ts (hunk 3 above)
Comment (igorbezsmertnyi) [reply packages/app/src/file-system/requests/get-paths.request.ts]: https://github.com/mudita/mudita-center/pull/907/commits/9c0bda2b2d75f8e396e1c22a11ac4efc41b4ef67

File: packages/app/src/file-system/dto/get-paths.input.ts (hunk 2 above)
Comment (igorbezsmertnyi) [reply packages/app/src/file-system/dto/get-paths.input.ts]: https://github.com/mudita/mudita-center/pull/907/commits/9c0bda2b2d75f8e396e1c22a11ac4efc41b4ef67

File: packages/app/src/file-system/constants/error.constant.ts (hunk 1 above)
Comment (igorbezsmertnyi) [reply packages/app/src/file-system/constants/error.constant.ts]: https://github.com/mudita/mudita-center/pull/907/commits/9c0bda2b2d75f8e396e1c22a11ac4efc41b4ef67

**Review by dkarski — APPROVED** [review]

**Review by wojcik92michal — APPROVED** [review]

**Pull request merged**